### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by amimem** [scspinney/pgm-fmri#6]
We need to search for good hyperparameter more thoroughly:
- Optimizer: SGD, Adam:
  - Learning rate, Momentum
  - b1,b2 for Adam
- Agreement Threshold:
  - Test more values around 0.9
- Geometric Mean
- Inverse Scaling of Gradients
